matplotcheck/tests/test_base.py

```diff
@@ -49,3 +49,73 @@ def test_options(pt_line_plt):
     ):
         pt_line_plt.assert_plot_type("foo")
     plt.close()
+
+
+""" assert_string_contains Tests """
+
+
+def test_assert_string_contains(pt_line_plt):
+    "Tests that assert_string_contains passes with correct expected strings."
```

Comment:
All of these docstrings (for this test and the ones below) should have three `"""` quotes around them.